### PR TITLE
resource_control: ru consistent with serverless

### PR DESCRIPTION
**Pull request by nolouch**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v7.2 (TiDB 7.2 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [x] v7.0 (TiDB 7.0 versions)
- [x] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by CabinfeverB — APPROVED** [review]

File: tidb-resource-control.md [hunk 1]
```diff
@@ -34,19 +34,7 @@ TiDB 资源管控特性提供了两层资源管理能力，包括在 TiDB 层的
...
-| CPU        | 1/3 RU / 毫秒  |
-| 读 IO      | 1/64 RU / KB |
-| 写 IO      | 1 RU / KB    |
-| 一次读请求的基本开销 | 0.25 RU      |
-| 一次写请求的基本开销 | 1.5 RU       |
-
-基于上表，假设某个资源组消耗的 TiKV 时间是 `c` 毫秒，`r1` 次请求读取了 `r2` KB 数据，`w1` 次写请求写入了 `w2` KB 数据，集群中非 witness TiKV 节点数是 `n`，则该资源组消耗的总 RU 的公式如下：
-
-`c`\* 1/3 + (`r1` \* 0.25 + `r2` \* 1/64) + (`w1` \* 1.5  + `w2` \* 1 \* `n`)
+更全面的 RU 阐述，可参考 [TiDB Cloud Serverless Tier Pricing](https://www.pingcap.com/tidb-cloud-serverless-pricing-details/)。**注意在 self-host 部署时，不包括其中网络和存储部分**。
```
Comment (hfxsd) [inline tidb-resource-control.md:37]:
```suggestion
更全面的 RU 阐述，可参考 [TiDB Cloud Serverless Tier Pricing](https://www.pingcap.com/tidb-cloud-serverless-pricing-details/)。**注意在 Self-Host 部署时，不会计量网络和存储费用**。
```

**Comment by nolouch** [comment]
ptal @hfxsd

File: tidb-resource-control.md [hunk 2]
```diff
@@ -32,21 +32,48 @@ TiDB 资源管控特性提供了两层资源管理能力，包括在 TiDB 层的
...
+    <thead>
+        <tr>
+            <th>Resource</th>
+            <th>RU Consumption</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td rowspan=3>Read</td>
+            <td>2 storage read batches 消耗 1 RU</td>
```
Comment (hfxsd) [inline tidb-resource-control.md:47]: 这个表里的英文改成中文吧？

File: tidb-resource-control.md [hunk 3]
```diff
@@ -32,21 +32,48 @@ TiDB 资源管控特性提供了两层资源管理能力，包括在 TiDB 层的
...
+            <td>1 KiB write request payload 消耗 1 RU</td>
+        </tr>
+        <tr>
+            <td>SQL CPU</td>
+            <td> 3 ms 消耗 1 RU</td>
+        </tr>
+    </tbody>
+</table>
+
+* 每个写操作都被复制到多个存储进程（3个用于没有索引的基于行的存储），并且每个重复被认为是一个不同的写操作。
```
Comment (hfxsd) [inline tidb-resource-control.md:70]:
```suggestion
* 每个写操作都被复制到多个存储进程（3 个用于没有索引的基于行的存储），并且每个重复被认为是一个不同的写操作。
```

File: tidb-resource-control.md (hunk 2 above)
Comment (nolouch) [reply tidb-resource-control.md:47]: 不太好改，有什么建议吗

File: tidb-resource-control.md [hunk 4]
```diff
@@ -32,21 +32,48 @@ TiDB 资源管控特性提供了两层资源管理能力，包括在 TiDB 层的
 
 ## 什么是 Request Unit (RU)
 
-Request Unit (RU) 是 TiDB 对 CPU、IO 等系统资源的统一抽象的单位, 目前包括 CPU、IOPS 和 IO 带宽三个指标。这三个指标的消耗会按照一定的比例统一到 RU 单位上。
+Request Unit (RU) 是 TiDB 对 CPU、IO 等系统资源的统一抽象的单位，是一种计量单位，用于表示对数据库的单个请求消耗的资源量。请求消耗的 RU 数量取决于多种因素，例如操作类型、或正在检索或修改的数据量。目前，RU 将包含以下资源的统计信息：
```
Comment (HuSharp) [inline tidb-resource-control.md:35]:
```suggestion
Request Unit (RU) 是 TiDB 对 CPU、IO 等系统资源的统一抽象的单位，是一种计量单位，用于表示对数据库的单个请求消耗的资源量。请求消耗的 RU 数量取决于多种因素，例如操作类型或正在检索或修改的数据量。目前，RU 将包含以下资源的统计信息：
```

File: tidb-resource-control.md [hunk 5]
```diff
@@ -32,21 +32,48 @@ TiDB 资源管控特性提供了两层资源管理能力，包括在 TiDB 层的
...
-| 写 IO      | 1 RU / KB    |
-| 一次读请求的基本开销 | 0.25 RU      |
-| 一次写请求的基本开销 | 1.5 RU       |
-
-基于上表，假设某个资源组消耗的 TiKV 时间是 `c` 毫秒，`r1` 次请求读取了 `r2` KB 数据，`w1` 次写请求写入了 `w2` KB 数据，集群中非 witness TiKV 节点数是 `n`，则该资源组消耗的总 RU 的公式如下：
-
-`c`\* 1/3 + (`r1` \* 0.25 + `r2` \* 1/64) + (`w1` \* 1.5  + `w2` \* 1 \* `n`)
+> **注意：**
+>
+> 上表只列举了 self-host 部署的计算 RU 时涉及的相关资源，其中不包括网络和存储部分。TiDB Cloud Serverless 的 RU 可参考 [TiDB Cloud Serverless Tier Pricing](https://www.pingcap.com/tidb-cloud-serverless-pricing-details/)。
```
Comment (hfxsd) [inline tidb-resource-control.md:76]:
```suggestion
> 上表只列举了自托管  (Self-Host) 部署的计算 RU 时涉及的相关资源，其中不包括网络和存储部分。TiDB Serverless 的 RU 可参考 [TiDB Serverless Pricing Details](https://www.pingcap.com/tidb-cloud-serverless-pricing-details/)。
```

File: tidb-resource-control.md [hunk 6]
```diff
@@ -32,21 +32,48 @@ TiDB 资源管控特性提供了两层资源管理能力，包括在 TiDB 层的
...
+            <td>1 KiB write request payload 消耗 1 RU</td>
+        </tr>
+        <tr>
+            <td>SQL CPU</td>
+            <td> 3 ms 消耗 1 RU</td>
+        </tr>
+    </tbody>
+</table>
+
+* 每个写操作都被复制到多个存储进程（3 个用于没有索引的基于行的存储），并且每个重复被认为是一个不同的写操作。
```
Comment (hfxsd) [inline tidb-resource-control.md:70]:
```suggestion
* 每个写操作都被复制到 3 个用于没有索引且基于行的存储进程。并且每次重复被认为是一个不同的写操作。
```

File: tidb-resource-control.md [hunk 7]
```diff
@@ -32,21 +32,48 @@ TiDB 资源管控特性提供了两层资源管理能力，包括在 TiDB 层的
 
 ## 什么是 Request Unit (RU)
 
-Request Unit (RU) 是 TiDB 对 CPU、IO 等系统资源的统一抽象的单位, 目前包括 CPU、IOPS 和 IO 带宽三个指标。这三个指标的消耗会按照一定的比例统一到 RU 单位上。
+Request Unit (RU) 是 TiDB 对 CPU、IO 等系统资源的统一抽象的单位，是一种计量单位，用于表示对数据库的单个请求消耗的资源量。请求消耗的 RU 数量取决于多种因素，例如操作类型或正在检索或修改的数据量。目前，RU 将包含以下资源的统计信息：
```
Comment (hfxsd) [inline tidb-resource-control.md:35]:
```suggestion
Request Unit (RU) 是 TiDB 对 CPU、IO 等系统资源的统一抽象的单位，是一种计量单位，用于表示对数据库的单个请求消耗的资源量。请求消耗的 RU 数量取决于多种因素，例如操作类型或正在检索或修改的数据量。目前，RU 包含以下资源的统计信息：
```

File: tidb-resource-control.md [hunk 8]
```diff
@@ -32,21 +32,48 @@ TiDB 资源管控特性提供了两层资源管理能力，包括在 TiDB 层的
...
+            <td>1 KiB write request payload 消耗 1 RU</td>
+        </tr>
+        <tr>
+            <td>SQL CPU</td>
+            <td> 3 ms 消耗 1 RU</td>
+        </tr>
+    </tbody>
+</table>
+
+* 每个写操作最终都被会复制到所有副本(TiKV 默认 3 数据副本)，并且每次复制都被认为是一个不同的写操作。
```
Comment (hfxsd) [inline tidb-resource-control.md:70]:
```suggestion
* 每个写操作最终都被会复制到所有副本（TiKV 默认 3 个数据副本），并且每次复制都被认为是一个不同的写操作。
```

**Review by hfxsd — APPROVED** [review]

File: tidb-resource-control.md [hunk 9]
```diff
@@ -32,21 +32,46 @@ TiDB 资源管控特性提供了两层资源管理能力，包括在 TiDB 层的
...
-| 一次写请求的基本开销 | 1.5 RU       |
-
-基于上表，假设某个资源组消耗的 TiKV 时间是 `c` 毫秒，`r1` 次请求读取了 `r2` KB 数据，`w1` 次写请求写入了 `w2` KB 数据，集群中非 witness TiKV 节点数是 `n`，则该资源组消耗的总 RU 的公式如下：
-
-`c`\* 1/3 + (`r1` \* 0.25 + `r2` \* 1/64) + (`w1` \* 1.5  + `w2` \* 1 \* `n`)
+> **注意：**
+>
+>- 每个写操作最终都被会复制到所有副本（TiKV 默认 3 个数据副本），并且每次复制都被认为是一个不同的写操作。
+>- 除了用户执行的查询之外，RU 还可以被后台任务消耗，例如自动统计信息收集。
+>- 上表只列举了自托管  (Self-Host) 部署的计算 RU 时涉及的相关资源，其中不包括网络和存储部分。TiDB Serverless 的 RU 可参考 [TiDB Serverless Pricing Details](https://www.pingcap.com/tidb-cloud-serverless-pricing-details/)。
```
Comment (shawn0915) [inline tidb-resource-control.md:74]: 自托管 (Self-Host) --> 是指 --> 本地部署 ?

Comment (shawn0915) [reply tidb-resource-control.md:74]: @nolouch pls take a look

File: tidb-resource-control.md [hunk 10]
```diff
@@ -32,21 +32,46 @@ TiDB 资源管控特性提供了两层资源管理能力，包括在 TiDB 层的
 
 ## 什么是 Request Unit (RU)
 
-Request Unit (RU) 是 TiDB 对 CPU、IO 等系统资源的统一抽象的单位, 目前包括 CPU、IOPS 和 IO 带宽三个指标。这三个指标的消耗会按照一定的比例统一到 RU 单位上。
+Request Unit (RU) 是 TiDB 对 CPU、IO 等系统资源的统一抽象的单位，是一种计量单位，用于表示对数据库的单个请求消耗的资源量。请求消耗的 RU 数量取决于多种因素，例如操作类型或正在检索或修改的数据量。目前，RU 包含以下资源的统计信息：
```
Comment (qiancai) [inline tidb-resource-control.md:35]:
```suggestion
Request Unit (RU) 是 TiDB 对 CPU、IO 等系统资源的统一抽象的计量单位，用于表示对数据库的单个请求消耗的资源量。请求消耗的 RU 数量取决于多种因素，例如操作类型或正在检索或修改的数据量。目前，RU 包含以下资源的统计信息：
```

File: tidb-resource-control.md [hunk 11]
```diff
@@ -32,21 +32,46 @@ TiDB 资源管控特性提供了两层资源管理能力，包括在 TiDB 层的
...
+    <thead>
+        <tr>
+            <th>资源类型</th>
+            <th>RU 消耗</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td rowspan=3>Read</td>
+            <td>2 storage read batches 消耗 1 RU</td>
```
Comment (qiancai) [inline tidb-resource-control.md:47]: 什么是 2 storage read batches？

File: tidb-resource-control.md (hunk 9 above)
Comment (qiancai) [inline tidb-resource-control.md:74]:
```suggestion
>- 上表只列举了本地部署的 TiDB 计算 RU 时涉及的相关资源，其中不包括网络和存储部分。TiDB Serverless 的 RU 可参考 [TiDB Serverless Pricing Details](https://www.pingcap.com/tidb-cloud-serverless-pricing-details/)。
```

File: tidb-resource-control.md (hunk 11 above)
Comment (shawn0915) [reply tidb-resource-control.md:47]: 从tidb serverless那个文章复制过来的吧，其实现在那个就已经通俗易懂了，为啥非要按照serverless的那个改呢？

File: tidb-resource-control.md (hunk 9 above)
Comment (nolouch) [reply tidb-resource-control.md:74]: 嗯

File: tidb-resource-control.md (hunk 11 above)
Comment (nolouch) [reply tidb-resource-control.md:47]: 和之前的不懂，这个是 batchs

File: tidb-resource-control.md [hunk 12]
```diff
@@ -32,21 +32,46 @@ TiDB 资源管控特性提供了两层资源管理能力，包括在 TiDB 层的
...
-| 一次写请求的基本开销 | 1.5 RU       |
-
-基于上表，假设某个资源组消耗的 TiKV 时间是 `c` 毫秒，`r1` 次请求读取了 `r2` KB 数据，`w1` 次写请求写入了 `w2` KB 数据，集群中非 witness TiKV 节点数是 `n`，则该资源组消耗的总 RU 的公式如下：
-
-`c`\* 1/3 + (`r1` \* 0.25 + `r2` \* 1/64) + (`w1` \* 1.5  + `w2` \* 1 \* `n`)
+> **注意：**
+>
+>- 每个写操作最终都被会复制到所有副本（TiKV 默认 3 个数据副本），并且每次复制都被认为是一个不同的写操作。
+>- 除了用户执行的查询之外，RU 还可以被后台任务消耗，例如自动统计信息收集。
+>- 上表只列举了本地部署的 TiDB 计算 RU 时涉及的相关资源，其中不包括网络和存储部分。TiDB Serverless 的 RU 可参考 [TiDB Serverless Pricing Details](https://www.pingcap.com/tidb-cloud-serverless-pricing-details/)。
```
Comment (qiancai) [inline tidb-resource-control.md:74]:
```suggestion
> **注意：**
>
> - 每个写操作最终都被会复制到所有副本（TiKV 默认 3 个数据副本），并且每次复制都被认为是一个不同的写操作。
> - 除了用户执行的查询之外，RU 还可以被后台任务消耗，例如自动统计信息收集。
> - 上表只列举了本地部署的 TiDB 计算 RU 时涉及的相关资源，其中不包括网络和存储部分。TiDB Serverless 的 RU 可参考 [TiDB Serverless Pricing Details](https://www.pingcap.com/tidb-cloud-serverless-pricing-details/)。
```

**Review by qiancai — COMMENTED** [review]
Rest LGTM

**Review by qiancai — APPROVED** [review]

**Pull request merged**